### ISSUE TITLE
Вопросы

### ISSUE THREAD

**Issue by sprutkam** [GabrielAgrela/OSROKBOT#1]
Доброе утро! 
Не совсем понимаю, что указывать в ANTIALIAS_METHOD=

**Comment by sprutkam** [comment]
Так, я понял. Проблема глубже. Я не знаю как создать файл .env ...

**Comment by sprutkam** [comment]
> pil.image has no attribute antialias

Установил pip install Pillow==9.5.0
Теперь при запуске OS_ROKBOT.py имею ответ "Your UI Instance" и больше ничего не происходит

**Comment by GabrielAgrela** [comment]
I developed the bot for the English version, that's probably the issue. Is the game client called " rise of kingdoms" in your version?

**Comment by sprutkam** [comment]
> I developed the bot for the English version, that's probably the issue. Is the game client called " rise of kingdoms" in your version?
Я догадался поменять интерфейс на английский; Клиент - rise of kingdoms"
